PowerPointLabs/PowerPointLabs/CropLab/CropOutPadding.cs

```diff
@@ -72,23 +74,29 @@ public static PowerPoint.ShapeRange Crop(PowerPoint.ShapeRange shapeRange)
                 float cropRatioTop = croppedImageRect.Top / (float)origImageRect.Height;
                 float cropRatioBottom = (origImageRect.Height - croppedImageRect.Height) / (float)origImageRect.Height;
 
-                float newCropLeft = origWidth * cropRatioLeft;
-                float newCropRight = origWidth * cropRatioRight;
-                float newCropTop = origHeight * cropRatioTop;
-                float newCropBottom = origHeight * cropRatioBottom;
+                float newCropLeft = Math.Max(origWidth * cropRatioLeft, cropLeft);
+                float newCropRight = Math.Max(origWidth * cropRatioRight, cropRight);
+                float newCropTop = Math.Max(origHeight * cropRatioTop, cropTop);
+                float newCropBottom = Math.Max(origHeight * cropRatioBottom, cropBottom);
+
+                if (!hasChange && (newCropLeft != cropLeft || newCropRight != cropRight || 
```

Comment:
The message doesn't seem to be showing up for one case:

1. Open CropOutPadding.pptx
2. Slide 8, click on one of the Gs and apply Crop Out Padding

These two values are different:
![image](https://cloud.githubusercontent.com/assets/19278089/24205998/f373bdea-0f57-11e7-960e-b896b63e730c.png)
Might want to use Epsilon based checking for floats